crates/project/src/config.rs

```diff
@@ -12,10 +12,14 @@ pub struct Config {
 
 impl Config {
     pub fn load(dir: PathBuf) -> io::Result<Config> {
-        let raw_config = fs::read_to_string(dir.join("aiken.toml"))?;
-
-        let config = toml::from_str(&raw_config).unwrap();
-
-        Ok(config)
+        if let Ok(raw_config) = fs::read_to_string(dir.join("aiken.toml")) {
+            let config = toml::from_str(&raw_config).unwrap();
+            Ok(config)
+        } else {
+            Err(io::Error::new(
```

Comment:
So what we could do so that we get the pretty errors is add a new variant to `crate::error::Error`. Probably could call it `MissingConfig`